app/package.json

```diff
@@ -31,10 +36,12 @@
     "@mui/material": "^5.8.6",
     "@reduxjs/toolkit": "^1.8.3",
     "formik": "^2.2.9",
+    "node": "^20.8.0",
```

Comment:
Can we make node and semver dev dependencies since they aren't used in the app? Not sure on the size of the node package but might be best to keep it out of the production build.

Reply:
Agreed. We should delete `node` and move `semver` to the dev dependencies.